data/product.d/ovirt.conf

```diff
@@ -26,3 +26,9 @@ req_partition_sizes =
 
 [User Interface]
 help_directory = /usr/share/anaconda/help/rhv
+
+[Security]
+
+# require user to set root password during oVirt installation
+root_password_must_be_set = True
```

Comment:
It feels like a UI thing. Modules don't enforce this rule, so I would keep it in the `[User Interface]` section.

Reply:
Changed.